includes/class-wc-countries.php

```diff
@@ -944,6 +944,16 @@ public function get_country_locale() {
 							'required' => false,
 						),
 					),
+					'GH' => array(
+						'postcode' => array(
+							'label'    => __( 'Postcode', 'woocommerce' ),
+							'required' => false,
+							'hidden'   => true,
+						),
+						'state'    => array(
+							'label' => __( 'State', 'woocommerce' ),
```

Comment:
Why you need to change this label?

Reply:
The state label? Again, just used another country that's already hidden as reference. Sorry if I did this wrong, first commit to the repo :/

Reply:
@claudiosanches I should have actually changed the label to 'Region'. I'll edit this now as well